BlazorAccessibilityTests.Tests/AriaAccordionTests.razor

```diff
@@ -1,335 +1,197 @@
+        var button1 = cut.Find("h3:nth-of-type(1) > button");
+        var button2 = cut.Find("h3:nth-of-type(2) > button");
 
         button2.Click();
 
         button1.GetAttribute("aria-expanded").Should().Be("false");
         button2.GetAttribute("aria-expanded").Should().Be("true");
-        panels[0].Instance.IsExpanded.Should().BeFalse();
-        panels[1].Instance.IsExpanded.Should().BeTrue();
     }
 
-    // aria-controls="ID"	button	
+    // aria-controls="ID" button	
     // Points to the ID of the panel which the header controls.
     [Fact(DisplayName = "aria-controls=ID Points to the ID of the panel which the header controls.")]
     public void AriaControlsAttribute()
     {
-        var cut = Render( @<AriaAccordion>
-                                <AccordionPanel Title="P1">
-                                    <p>P1 Content</p>
-                                </AccordionPanel>
-                           </AriaAccordion>);
-        
-        var panel = cut.FindComponent<AccordionPanel>();
-        var buttonId = panel.Find("button").GetAttribute("aria-controls");
-        
-        panel.Find("div")
-            .GetAttribute("id")
-            .Should().Be(buttonId);
-            
-    }
-
-
-// aria-disabled="true"	button	
-// If the accordion panel is expanded and is not allowed to be collapsed, then set to true.
-
-    [Fact(DisplayName = "If the accordion panel is expanded and is not allowed to be collapsed, then set to true.")]
-    public void AriaDisabledAttribute()
-    {
-        var cut = Render(@<AriaAccordion>
-                                <AccordionPanel Title="P1">
-                                    <p>P1 Content</p>
-                                </AccordionPanel>
-                                <AccordionPanel Title="P2">
-                                    <p>P2 Content</p>
-                                </AccordionPanel>
-                        </AriaAccordion>
-    );
-
-        var panels = cut.FindComponents<AccordionPanel>();
-        var button1 = panels[0].Find("button");
-        var button2 = panels[1].Find("button");
-
-        button1.HasAttribute("aria-disabled").Should().BeTrue();
-        button2.HasAttribute("aria-disabled").Should().BeFalse();
-    }
-
-// region		div	
-// Creates a landmark region that contains the currently expanded accordion panel.
-
-    [Fact(DisplayName = "Creates a landmark region that contains the currently expanded accordion panel.")]
-    public void RegionAttribute()
-    {
-        var cut = Render(@<AriaAccordion>
-                                <AccordionPanel Title="P1">
-                                    <p>P1 Content</p>
-                                </AccordionPanel>
-                                <AccordionPanel Title="P2">
-                                    <p>P2 Content</p>
-                                </AccordionPanel>
-                        </AriaAccordion>
-    );
+        RenderTree.Add<AriaAccordion>();
```

Comment:
This adds the `AriaAccordion` to the render tree above whatever is passed to the `Render` method. So the component referenced by `cut` will be the `AccordionPanel`, but it will be rendered inside an `AriaAccordion` component.

In other words, the following:

```c#
RenderTree.Add<AriaAccordion>();
var cut = Render(@<AccordionPanel Title="P1">CONTENT</AccordionPanel>);
```

So this is similar to writing the following code:

```c#
var accordion = Render(@<AriaAccordion><AccordionPanel Title="P1">CONTENT</AccordionPanel>);
var cut = accordion.FIndComponent<AccordionPanel>();
```

Not fewer lines, but a bit more explicit about what is being rendered.